Loop/Managers/ServicesManager.swift

```diff
@@ -23,6 +23,8 @@ class ServicesManager {
 
     private let servicesLock = UnfairLock()
 
+    weak var dataManager: LoopDataManager?
```

Comment:
Curious—what's this addition for?

Reply:
This is from not noticing that while resolving merge conflicts, removing! Thanks for catching that